### ISSUE TITLE
Accessibility: it is not possible to navigate to attribution dialog close button

### ISSUE THREAD

**Issue by ronnymikalsen** [NationalLibraryOfNorway/ngx-mime#259]
**Describe the bug**
Attribution dialog is missing arial-label on close button

**To Reproduce**

1.  Go to http://localhos:4200/demo?manifestUri=assets%2Ffixtures%2Fpresentation%2F2%2Fdifferent-size-manifest.json'
2.  Try to navigate to close button i attributon

**Expected behavior**
Focus should be in attribution dialog, and it should be possible to navigate to close button

**Which versions of NGX-Mime, Angular, Material, OS, TypeScript, browsers are affected?**
All
